.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
```

Comment:
Perhaps this is the same as before.... I just wonder if we should not use the DHIS2 ESLint rules here?